src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClient/SqlConnection.cs

```diff
@@ -153,15 +153,27 @@ public SqlRetryLogicBaseProvider RetryLogicProvider
         [ResDescription(StringsHelper.ResourceNames.TCE_SqlConnection_TrustedColumnMasterKeyPaths)]
         public static IDictionary<string, IList<string>> ColumnEncryptionTrustedMasterKeyPaths => _ColumnEncryptionTrustedMasterKeyPaths;
 
+        internal SqlConnection(string connectionString, bool useExperimental) : this()
+        {
```

Comment:
```suggestion
        {
            _isExperimental = useExperimental;
```
?

Reply:
Thanks! I had reworked these a bit and accidentally dropped this.